platform-includes/getting-started-verify/node.mdx

```diff
@@ -1,16 +1,16 @@
 ```javascript
-const transaction = Sentry.startTransaction({
+Sentry.startSpan({
   op: "test",
   name: "My First Test Transaction",
+}, () => {
+  setTimeout(() => {
+    try {
+      foo();
+    } catch (e) {
+      Sentry.captureException(e);
+    } finally {
+      transaction.finish();
```

Comment:
m: This shouldn't be here anymore, no? Generally, would the `setTimeout` semantic still work with `startSpan`? I don't think so